app/src/main/java/in/testpress/testpress/ui/ItemListFragment.java

```diff
@@ -177,6 +186,11 @@ public boolean onOptionsItemSelected(final MenuItem item) {
         }
     }
 
+    private void getProducts(){
```

Comment:
This is wrong. ItemListFragment is an independent abstract class. Concrete functions like these should be implemented in the class which inherits this.